tool/lib/utils.dart

```diff
@@ -88,13 +88,27 @@ class CliCommand {
     );
   }
 
+  /// CliCommand helper for running git commands.
+  ///
+  /// Arguments can be passed in as a single string, this helper will split them
+  /// up using their spaces. e.g. CliCommand.git(['checkout test-branch'])
+  ///
+  /// If you don't want the args to be split by their spaces then you can set
+  /// [split] to false.
   factory CliCommand.git(
-    String args, {
+    List<String> args, {
```

Comment:
wdyt about instead having two parameters: 
```
factory CliCommand.git({
  String? cmd,
  List<String>? args,
  bool throwOnException = true,
}) : assert((cmd == null) != (args == null), 'Only one of `cmd` and `args` may be specified) {
...
}
```

I worry that it isn't intuitive to be passing a single string to a list in the most common case. Having named parameters would make this more clear.

Reply:
I dig it!